recipes/pyopengl/post-link.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+$PYTHON -c "import OpenGL.GL" && echo OpenGL OK || echo Warning: Missing OpenGL driver, install with yum install mesa-libGL-devel or equivalent
```

Comment:
Would this suppress the exit status? It would be nice if this didn't install without OpenGL present. That will only happen if we have a non-zero exist status.

Also, this is only necessary for Linux so a `uname` check would be great here to avoid this on Mac.